images/plbase/plbase-install.sh

```diff
@@ -27,7 +28,7 @@ yum -y install \
     tar \
     ImageMagick \
     fontconfig-devel \
-    texlive \
+    texlive-base \
```

Comment:
While I agree this is probably safe, I'm a bit worried about how to test it. I guess it will get some testing on canary servers, but is this sufficient?

Reply:
Installing only texlive-base and texlive-dvipng doesn't seem to install any actual tex/latex compiler. (The yum texlive-base seems very different from apt texlive-base...)

Reply:
@mfsilva22 it looks like this was added for your course, can you provide an example question or two that uses LaTeX formatting in matplotlib labels?

Reply:
@echuber2 good catch, after testing with the `demo/annotated/LectureVelocity/1-Introduction` question, it looks like matplotlib does indeed literally just shell out to `latex`, which is a problem because `texlive-base` doesn't include that binary

I spent some time trying to figure out which subset of `texlive-*` packages are necessary, but I kept having to add more and more, and eventually I was almost back at the number of MB that `texlive` itself would install 😅

I'll back this change out; it's a shame that latex on RHEL is such a pain.